.gemspec

```diff
@@ -37,19 +37,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.rubyforge_project = "puppet"
   s.summary = "Puppet, an automated configuration management tool"
-
-  if s.respond_to? :specification_version then
-    s.specification_version = 3
-
-    if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<facter>, [">= 1.7", "< 3"])
-      s.add_runtime_dependency(%q<hiera>, [">= 2.0", "< 4"])
-    else
-      s.add_dependency(%q<facter>, [">= 1.7", "< 3"])
-      s.add_dependency(%q<hiera>, [">= 2.0", "< 4"])
-    end
-  else
-    s.add_dependency(%q<facter>, [">= 1.7", "< 3"])
-    s.add_dependency(%q<hiera>, [">= 2.0", "< 4"])
-  end
+  s.specification_version = 3
+  s.add_runtime_dependency(%q<facter>, [">= 1.7", "< 3"])
```

Comment:
No, but that's unrelated to dropping support for older rubygems, so I was thinking that would be a separate PR.